rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/webapp/PortalConfiguration.java

```diff
@@ -155,7 +156,12 @@ public static String getAbsolutePath(String userPath) {
         if (new File(userPath).isAbsolute()) {
             return userPath;
         } else {
-            return PortalConfiguration.REST_HOME.getValueAsString() + File.separator + userPath;
+            File pathName = new File(PortalConfiguration.REST_HOME.getValueAsString(), userPath);
```

Comment:
This code is appearing in several places for the same logic. Don't you want to extract it?